src/DIRAC/WorkloadManagementSystem/ConfigTemplate.cfg

```diff
@@ -19,6 +19,21 @@ Services
     }
   }
   ##END
+  TornadoPilotLogging
```

Comment:
But before this line a 

```
##BEGIN TornadoPilotLogging
```

and use it for the documentation in TornadoPilotLoggingHandler.py

Reply:
Done